src/web-ui/src/public/components/Product.vue

```diff
@@ -1,18 +1,15 @@
 <template>
-  <div class="col-sm-12 col-md-4 col-lg-3 mb-4">
+  <div class="featured-product p-3 d-flex flex-column justify-content-between text-left">
     <!-- Card -->
     <router-link class="link" :to="{name:'ProductDetail', params: {id: product.id}, query: {feature: feature, exp: experimentCorrelationId}}">
-      <div class="card mx-auto h-100 product-card text-left">
+      <div>
         <img :src="productImageURL" class="card-img-top" :alt="product.name">
-        <div class="card-body">
-          <h5 class="card-title">{{ product.name }}</h5>
-          <p class="card-text">{{ product.description }}</p>
-          <five-stars />
-          <p class="card-text">${{ product.price.toFixed(2) }}</p>
-        </div>
-        <div class="card-footer" v-if="experiment">
-          <small class="text-muted"><i class="fa fa-balance-scale"></i> {{ experimentDescription }}</small>
-        </div>
+        <div class="product-name">{{ product.name }}</div>
+        <FiveStars />
+        <div>${{ product.price.toFixed(2) }}</div>
+      </div>
+      <div v-if="experiment" class="experiment mt-1 d-flex align-items-center text-muted">
+        <i class="fa fa-balance-scale mr-2"></i> {{ getExperimentDescription(experiment) }}
```

Comment:
`experimentDescription` is a computed property. Does the call to `getExperimentDescription()` passing the experiment work? Did you test with an active experiment?

Reply:
I think this was the result of some copy-paste from the `ProductCarousel` component. It should indeed be using `experimentDescription` here.